xbmc/music/MusicDatabase.cpp

```diff
@@ -10823,8 +10869,7 @@ void CMusicDatabase::SetPropertiesFromAlbum(CFileItem& item, const CAlbum& album
   if (album.iVotes > 0)
     item.SetProperty("album_votes", album.iVotes);
 
-  if (album.bBoxedSet)
-    item.SetProperty("album_bBoxedSet", true); //! @todo Set listItem the same way as totalDiscs ??
+  item.SetProperty("album_isboxset", album.bBoxedSet); //! @todo Set listItem the same way as totalDiscs ??
```

Comment:
Remove "todo" comment

Reply:
Properties set in SetPropertiesFromAlbum are for 2 purposes:
a) Making album data available to skins (while showing an item list of songs) via `Listitem.property(Album_XXX)` 
b) Holding album data that does not fit in `CMusicInfoTag` e.g. theme
Hence some album data ends up in both Item.musicInfoTag and a property e.g. album title.

JSON gets data that has been **serialized**  from item musicInfoTag and from properties. It looks in musicInfoTag first, and only if it doesn't find the field there does it look at properties. So for example although there is an "album_title" property, JSON gets album title from "title" created by `CMusicInfoTag::Serialize`.

Hence there are code changes needed elsewhere:
1) Documentation of GUI properties for Album_Totaldiscs (property actually already created by your previous PR) and Album_Isboxset (quietly renaming the property that none knew about), adding to GUIInfoManager.cpp
```
///   \table_row3{   <b>`ListItem.Property(Album_Totaldiscs)`</b>,
///                  \anchor ListItem_Property_Album_Totaldiscs
///                  _string_,
///     @return The total number of discs belonging to an album.
///     <p><hr>
///     @skinning_v19 **[New Infolabel]** \link ListItem.Property(Album_Totaldiscs) `ListItem.Property(Album_Totaldiscs)`\endlink
///     <p>
///   }
///   \table_row3{   <b>`ListItem.Property(Album_Isboxset)`</b>,
///                  \anchor ListItem_Property_Album_Isboxset
///                  _string_,
///     @return **True** if the album is a boxset.
///     <p><hr>
///     @skinning_v19 **[New Infobool]** \link ListItem.Property(Album_Isboxset) `ListItem.Property(Album_Isboxset)`\endlink
///     <p>
///   }
```

2). Serialize the fields previously added to CMusicInfoTag by adding
```
  value["isboxset"] = m_bBoxset;
  value["totaldiscs"] = m_iDiscTotal;
  value["disctitle"] = m_strDiscSubtitle;
```